backend/internal/handlers/classrooms/assignments/assignments.go

```diff
@@ -66,6 +69,80 @@ func (s *AssignmentService) createAssignment() fiber.Handler {
 	}
 }
 
+
+
+func (s *AssignmentService) acceptAssignment() fiber.Handler {
+	return func(c *fiber.Ctx) error {
+		var assignment models.AssignmentAcceptRequest
+		err := c.BodyParser(&assignment)
+		if err != nil {
+			return errs.InvalidRequestBody(models.AssignmentOutline{})
+		}
+
+		client, err := middleware.GetClient(c, s.store, s.userCfg)
+
+		if err != nil {
+			return errs.AuthenticationError()
+		}
+
+		username, err := client.GetCurrentUser(c.Context())
+		if err != nil {
+			return errs.GithubAPIError(err)
+		}
+
+		forkName := assignment.SourceRepoName + "-" + strings.ReplaceAll((username.Login), " ", "")
+
+		studentwork := createMockStudentWork(forkName, assignment.AssignmentName, int(assignment.AssignmentID))
+
+
+		err = s.store.CreateStudentWork(c.Context(), &studentwork, username.ID)
+		if err != nil {
+			return errs.InternalServerError()
+		}
+
+		err = client.ForkRepository(c.Context(), assignment.OrgName, assignment.OrgName, assignment.SourceRepoName, forkName)
+		if err != nil {
+			return errs.InternalServerError()
+		}
+
+		c.Status(http.StatusOK)
```

Comment:
this could be combined into just return c.Status(http.StatusOK)

Reply:
Actually cannot- c.Status returns a Ctx, and the function expects an error

In other functions we do c.Status().json(body), which is okay because .json() returns an error

Reply:
ah, I see. Maybe we can just add a message to be more uniform with everywhere else in our codebase?